src/tests/JIT/Regression/JitBlue/Runtime_61940/Runtime_61940.csproj

```diff
@@ -0,0 +1,19 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <Optimize>True</Optimize>
+    <CLRTestBatchPreCommands><![CDATA[
+      $(CLRTestBatchPreCommands)
+      set DOTNET_JitNoStructPromotion=1
+      set DOTNET_JitNoCSE=1
+      ]]></CLRTestBatchPreCommands>
+    <BashCLRTestPreCommands><![CDATA[
+      $(BashCLRTestPreCommands)
+      export DOTNET_JitNoStructPromotion=1
+      export DOTNET_JitNoCSE=1
+      ]]></BashCLRTestPreCommands>
```

Comment:
Does this really need `JitNoStructPromotion` and `JitNoCSE` - seems like a copy & paste error?

Also, I don't think this should be `Jit/Regression` folder, that is meant for tests for bug fixes.

`JIT\opt\OptimizeBools` would be a better place.